bin/ruby-build

```diff
@@ -566,8 +566,16 @@ build_package_standard() {
       $CONFIGURE_OPTS ${!PACKAGE_CONFIGURE_OPTS} "${!PACKAGE_CONFIGURE_OPTS_ARRAY}" || return 1
   ) >&4 2>&1
 
+
   { "$MAKE" $MAKE_OPTS ${!PACKAGE_MAKE_OPTS} "${!PACKAGE_MAKE_OPTS_ARRAY}"
-    "$MAKE" install $MAKE_INSTALL_OPTS ${!PACKAGE_MAKE_INSTALL_OPTS} "${!PACKAGE_MAKE_INSTALL_OPTS_ARRAY}"
+  } >&4 2>&1
+
+  if [ "$package_name" == "ruby-trunk" ]; then
+    { "$MAKE" update-gems
```

Comment:
If I remember correctly, it's also necessary to execute `"$MAKE" extract-gems` to use bundled gems when manually building ruby, isn't it? Not sure, just confirmation.